db_diff/tskdbdiff.py

```diff
@@ -285,10 +286,16 @@ def _dump_output_db_bb(db_file, bb_dump_file, isMultiUser, pgSettings, id_obj_pa
                             elif attr["value_type"] == 4:
                                 attr_value_as_string = "bytes"                        
                             elif attr["value_type"] == 5:
-                                attr_value_as_string = str(attr["value_int64"])                        
+                                attr_value_as_string = str(attr["value_int64"])
+                            elif attr["value_type"] == 6:
+                                attr_value_as_string = str(attr["value_text"])
                             if attr["display_name"] == "Associated Artifact":
                                 attr_value_as_string = TskDbDiff._get_associated_artifact_type(attribute_cursor, attr_value_as_string, isMultiUser)
                             patrn = re.compile("[\n\0\a\b\r\f]")
+                            if attr_value_as_string == None:
```

Comment:
I doubt it matters too much, but I think `is None` and `is not None` is the preferred way to check for `None` based on [this](https://stackoverflow.com/a/3965129) and [this](https://stackoverflow.com/a/2020606).